sessionctx/variable/sysvar.go

```diff
@@ -1692,6 +1692,14 @@ var defaultSysVars = []*SysVar{
 		s.RcReadCheckTS = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBPointLockReadUseLastTso, Type: TypeBool, Value: BoolToOnOff(DefTiDBPointLockReadUseLastTso), SetSession: func(s *SessionVars, val string) error {
+		s.RcPointLockReadUseLastTso = TiDBOptOn(val)
+		return nil
+	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBInsertUseLastTso, Type: TypeBool, Value: BoolToOnOff(DefTiDBInsertUseLastTso), SetSession: func(s *SessionVars, val string) error {
+		s.RcInsertUseLastTso = TiDBOptOn(val)
+		return nil
+	}},
```

Comment:
The users could not know how to config these variables as they're pretty complex and detailed about the kernel implementation. Perhaps a single variable like `tidb_rc_write_check_ts` is more appropriate and it takes not that much to learn, the scope of its effect could be described in the document.

Besides, why should it be limited to `PointGetExecutor`? I remember `RcCheckTS` dost not have such constraints.

Reply:
1. tidb_rc_write_use_last_ts
We itegrates these two variables tidb_rc_insert_use_last_tso, tidb_rc_point_lock_read_use_last_tso to one which named tidb_rc_write_use_last_ts, and there are two ways to define the variable tidb_rc_write_use_last_ts.

1.1 tidb_rc_write_use_last_ts = "insert","point-write", "insert" | "point-write"
tidb_rc_write_use_last_ts can control insert, point-delete, point-update, point-lock-read whether don't get ts from PD at first precisely.
But it is set not so Conveniently. Users will lookup the system variable explanation every time, because nobody can remember it precisely.

1.2 tidb_rc_write_use_last_ts = ON | OFF
It enables or disables wheter to get ts from PD at first together for insert, point-delete, point-update, point-lock-read. It's not so 
flexible. But It's so easy for users to set this variable.

2. tidb_rc_use_last_ts
We itegrates these three variables tidb_rc_insert_use_last_tso, tidb_rc_point_lock_read_use_last_tso, tidb_rc_read_check_ts to one which named tidb_rc_use_last_ts.
tidb_rc_use_last_ts = "read" , "insert", "point-write", any combinations of "read", "insert", "point-write" 
The benifits is here are only one variables, but it makes code more complex and it's difficulty to set the value for users, nobody can remember it precisely. Another disadvantage is bring compatible problem if tidb_rc_read_check_ts is already used for some users.

I recommend option 1.2

Reply:
> Besides, why should it be limited to PointGetExecutor? 

Like RR isolation, TiDB doesn't get the latest ts from PD as for_update_ts but use the last ts of this transaction for point-lock-read, point-update, point-delete, because they access only one row and decrease the conflict risk to very low.